sticht/rollbacks/metrics.py

```diff
@@ -24,7 +24,7 @@ def _get_metric_configs_for_service_by_cluster(
 ) -> Dict[str, Dict[str, Any]]:  # TODO: add type for rollback file config
     configs = {}
     for filename in get_rollback_files_from_soaconfigs(soa_dir, service=service):
-        with open(filename, 'r') as file:
+        with open(filename) as file:
```

Comment:
Any reason for why we need to remove read-only when opening this file?

Reply:
pre-commit did this - likely because one of the hooks we have prefers that we don't re-state defaults?